_events/sr2021/knockouts.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Competition Knockouts and Final
+date: 2021-05-01 13:00:00
```

Comment:
Given these events (league and competition) should be shorter than kickstart, i'd be interested in pushing these slightly later (14:00?). Especially useful as we'll be pre-recording events, so need as much time as possible.

SR2020 was run at 15:00: https://studentrobotics.org/events/sr2020/virtual-competition-knockouts/

Reply:
So here's an interesting thought -- is there a time we can run them so that the Australians:
a) don't get too much benefit from being 10h ahead, but
b) can watch them live?

Reply:
For example, 9am UTC would be mid-evening in Adelaide. If we shut off entries at 9pm UTC (7:30am Adelaide) then everyone can watch live and the time difference doesn't help the Australians unduly.

Alternatively, we could do something like saying that the submission cut-off was "Friday midnight in your local timezone". That separates the submission timing from the livestream timing.

I'm not sure there's a solution where everyone can watch them live and still have it at a comfortable hour though.

Separately, I am seriously hoping that match running and scoring will be fully automated this year. That should reduce a lot of the concerns around needing time to record things.

Reply:
An alternative here would be to not specify the time for now and sort this out nearer the time? I think the value of getting these out this weekend is greater than getting this perfect now.

Reply:
https://github.com/srobo/website/pull/240/commits/201f7d372a853db106258a6a632add33f40abb34 sidesteps this by not setting times for now.

Reply:
-> https://github.com/srobo/tasks/issues/643